predictor_robojudge.ipynb

```diff
@@ -569,14 +559,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ranking_df[ranking_df.CountryName == \"Italy\"]"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
```

Comment:
I think the below cell "Specific country (group by)" is no longer needed, since the metrics have already been computed. @ofrancon can you confirm?

Reply:
Yes not needed anymore, that was just some early test